sites/examplesshowcase/README.md

```diff
@@ -1,3 +1,12 @@
 A simple web app that showcases various examples of GopherJS React copmonents.
 
+To run this web app locally:
+
+```bash
+npm install
+gopherjs serve -w
```

Comment:
Thanks @shurcooL - it's one of those things that is muscle memory, nothing else. Will switch all instances now.